src/report/report.py

```diff
@@ -41,16 +42,18 @@ def __init__(self, firewatch_config: Configuration, job: Job) -> None:
             else:
                 exit(0)
 
+        bugs_filed = None
+        bugs_updated = None
         # If job has failures, file bugs
         if job.has_failures:
-            bugs_filed = self.file_jira_issues(
+            bugs_filed, bugs_updated = self.file_jira_issues(
                 failures=job.failures,  # type: ignore
                 firewatch_config=firewatch_config,
                 job=job,
             )
             if len(bugs_filed) > 1:
                 self.relate_issues(issues=bugs_filed, jira=firewatch_config.jira)
-        else:
+        if (bugs_filed is None or len(bugs_filed) == 0) and (bugs_updated is None or len(bugs_updated) == 0):
```

Comment:
```suggestion
        if not bugs_filed and not bugs_updated:
```

Reply:
Change not required, just a suggestion for readability. Tested change locally with all passing.